test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -908,11 +910,6 @@ TEST(mongoRegisterContext_update, NotifyContextAvailability1)
           .Times(0);
   setNotifier(notifierMock);
 
-  TimerMock* timerMock = new TimerMock();
```

Comment:
utInit() creates the TimerMock. Not sure what would happen if we had two timer mocks ...